### DISCUSSION TITLE
Retrieving a completion with reference id

### DISCUSSION THREAD

**Discussion by iamprazol** [openai-php/client#118] [Q&A] [answered]
How can I retrieve a completion that has been finished with a reference id?

**Comment by pb30** [comment, marked as answer]
There's no retrieval methods on API for completions. https://platform.openai.com/docs/api-reference/completions

I'd guess they return an ID just for internal troubleshooting/monitoring purposes.

**Comment by plamenradev** [comment]
https://platform.openai.com/docs/api-reference/chat/get
Will be good to have it

**Reply by iBotPeaches** [reply]
This is a thread about Completion though and you are posting about Chat. Those are 2 different endpoints - as this Q/A stands its not possible for Completion, but possible for Chat. I do see we are missing the get endpoint for Chat though
